src/app-pages/map/components/location-details/components/time-series/TimeSeriesSection.js

```diff
@@ -28,7 +28,8 @@ const TimeSeriesSection = ({
   }
 
   const layout = {
-    width: "100%",
+    autosize: true,
```

Comment:
I tried using autosize, but it causes a bug where if you're on the location page and manually shrink your browser's width and then make your browser full screen again, the plot will resize to take up the full width of the page and push the time series table below it. It might be better to have the plot take up the full width and have the time series table below it, but this autosize causes the behavior to be inconsistent.

Reply:
Yes I think the unpredictability of using flex grow (so there is no real defined width) is part of the problem. Plotly does some internal work to inspect the width of the parent and set a specific pixel width to the inner plot (you can see this in the element inspector). So maybe we want to take a more programmatic approach to setting the width, depending on whether it is in sidebar or full screen? e.g. if in sidebar, width is 100%, if in full screen, width is 50%, etc. That *may* help let plotly more correctly determine the width it should use.

Reply:
In other words, instead of using flex-grow, we would set specific width depending on the mode (sidebar vs full). What do you think?

Reply:
Yea I think that could work